scripts/Muon/GUI/FrequencyDomainAnalysis/FFT/fft_presenter.py

```diff
@@ -3,25 +3,35 @@
 
 import mantid.simpleapi as mantid
 
-from Muon import thread_model
+from Muon.GUI.Common import thread_model
 
 
 class FFTPresenter(object):
+
     """
     This class links the FFT model to the GUI
     """
-    def __init__(self,view,alg,load):
-        self.view=view
-        self.alg=alg
-        self.load=load
+
+    def __init__(self, view, alg, load):
+        self.view = view
+        self.alg = alg
```

Comment:
Why `alg` and not `FFTmodel` or something similar?

Reply:
its actually a thread model